nexus/src/external_api/views.rs

```diff
@@ -20,6 +20,20 @@ use std::net::IpAddr;
 use std::net::SocketAddrV6;
 use uuid::Uuid;
 
+/// Wrapper for unpaginated lists of items
+#[derive(Debug, Clone, Deserialize, Serialize, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+#[schemars(rename = "{}List")]
+pub struct List<T> {
+    pub items: Vec<T>,
+}
+
+impl<T> From<Vec<T>> for List<T> {
+    fn from(items: Vec<T>) -> Self {
+        Self { items }
+    }
+}
+
```

Comment:
This is nice and generic, though there are so few of these unpaginated things that I wonder if it makes sense to make one-offs with an appropriately named key, like  

```rs
struct ExternalIps {
  ips: Vec<ExternalIp>,
}
```

like in #1164. Either way we should standardize on one approach or the other.

Reply:
Yeah, I'm inclined to have all list types be formatted the same just for ease of implementation. It reduces the delta between a paginated endpoint and a non-paginated endpoint.

Reply:
In that case maybe I should call this `Results` to go with `ResultsPage`...